37_environment_lab/aliases

```diff
@@ -0,0 +1,9 @@
+alias ll='ls -l'
+alias rm='rm -i'
+alias cp='cp -i'
+alias count_files='ls |wc -l'
+alias bysize='ls -l -S'
+alias ff='ls |grep '$1'
+alias lsf='ls -l |grep ^-'
+alias lsd=ls -l |grep ^d'
+alias lsg='ls -l |sort' 
```

Comment:
Re. the last one - You can use the other 2 aliases just defined to get a nicer result: `alias lsg=lsd; lsf`